src/common/evaluation/QA/eval_depthmap_models/src/evaluate.py

```diff
@@ -293,6 +293,11 @@ def get_prediction(model_path: str, dataset_evaluation: tf.data.Dataset) -> np.a
     print(f"Calculate and save the results to {csv_file}")
     utils.calculate_and_save_results(df_grouped, EVAL_CONFIG.NAME, csv_file,
                                      DATA_CONFIG, RESULT_CONFIG, fct=calculate_performance)
+
+#     accuracy_df = utils.filter_scans(df_grouped,ACC)
+    sample_csv_file = f"{OUTPUT_CSV_PATH}/{RUN_ID}_scans.csv"
```

Comment:
Please prefix with `scans_` to be consistent

```suggestion
    sample_csv_file = f"{OUTPUT_CSV_PATH}/scans_{RUN_ID}.csv"
```